base/uk.ac.stfc.isis.ibex.epics/src/uk/ac/stfc/isis/ibex/epics/pvmanager/PVManagerSettings.java

```diff
@@ -0,0 +1,31 @@
+package uk.ac.stfc.isis.ibex.epics.pvmanager;
+
+import java.io.IOException;
+import org.eclipse.core.runtime.FileLocator;
+
+import uk.ac.stfc.isis.ibex.logger.IsisLog;
+
+public class PVManagerSettings {
```

Comment:
Checkstyle issue: `Utility classes should not have a public or default constructor.`